src/frontend/screens/MapScreen/track/TrackPathLayer.tsx

```diff
@@ -3,13 +3,13 @@ import {usePersistedTrack} from '../../../hooks/persistedState/usePersistedTrack
 import * as React from 'react';
 import {StyleSheet} from 'react-native';
 import {LineString} from 'geojson';
-import {useLocation} from '../../../hooks/useLocation';
 import {LocationHistoryPoint} from '../../../sharedTypes/location';
+import {useLocation} from '../../../hooks/useLocation';
 
 export const TrackPathLayer = () => {
   const locationHistory = usePersistedTrack(state => state.locationHistory);
   const isTracking = usePersistedTrack(state => state.isTracking);
-  const {location} = useLocation({maxDistanceInterval: 3});
+  const {location} = useLocation();
```

Comment:
You will need some kind of filter on here or this is going to be a battery hog with very frequent map updates (location updates can come in at 60 times a second I think on some phones)

Reply:
https://github.com/digidem/comapeo-mobile/pull/387/commits/fbedb98d066c2fdf3a3b4bd4f6fcade8d1a9a834 I changed the default (i chose completely arbitrary values, but it should throttle it now)